src/content/1.7/development/webservice/_index.md

```diff
@@ -13,13 +13,13 @@ PrestaShop enables merchants to give third-party tools access to their shop's da
 The PrestaShop web service uses the REST architecture in order to be available on as many platforms as possible, since the HTTP protocol and XML files are understood by most platforms, if not all.
 
 {{% notice note %}}
-[CRUD](http://en.wikipedia.org/wiki/Create,_read,_update_and_delete)
+[CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete)
```

Comment:
As these examples are used for development, it is likely that someone working on its webservice is using localhost and consequently does not have https enabled. Maybe `https` is not relevant here ?